Khan/perseus
# [Interactive Graph + Editor] Add aria label and description to entire interactive graph and its UI to the editor

## Summary:
- Add `fullGraphAriaLabel` and `fullGraphAriaDescription` fields to interactive graph.
- Hook up the graph so that if the `fullGraphAriaLabel` and/or `fullGraphAriaDescription` fields are present, then the graph has the respective `aria-label` and `aria-describedby` properties.
- Add the UI to the interactive graph editor that allows the content author to add a title (label) and description to the graph.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2283

## Test plan:
`yarn jest packages/perseus-editor/src/components/__tests__/interactive-graph-description.test.tsx`
`yarn jest packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor.test.tsx`
`yarn jest packages/perseus/src/widgets/interactive-graphs/interactive-graph.test.tsx`

Storybook
- Go to http://localhost:6006/iframe.html?id=perseuseditor-widgets-interactive-graph--interactive-graph-with-aria-label&viewMode=story
- There should already be a title and description written in their respective fields
- Use a screenreader to tab to the graph
- Confirm that the title and description are read. They will both be read at once in Safari and Firefox. With Chrome, use the screenreader controls to go to the next element to hear the description.
- Change the values in the title and description inputs, and confirm that the aria label and description on the graph updates.

## Demo:

https://github.com/user-attachments/assets/3c825e87-a989-4327-8319-2205e8913b4e

NOTE! The janky textarea will be replaced with a Wonder Blocks TextArea [soon](https://khanacademy.atlassian.net/browse/LEMS-2332).

![image](https://github.com/user-attachments/assets/a4b6c931-4619-4c71-bb35-542cea6817fd)
